test/browser/specs/custom-base.test.mjs

```diff
@@ -0,0 +1,15 @@
+import assert from 'node:assert'
+import test from 'node:test'
+import { execa } from 'execa'
+
+test('custom-base', async () => {
+  const result = await execa('npx', [
+    'vitest',
+    'run',
```

Comment:
Was this using `npx` before? Signal terminals are buggy when scripts are launched via `npx`. I can't find the link for the issue or older references for these issues right now. Consider using `startVitest` instead.

Reply:
`pnpm exec` should also work correctly.

Reply:
Currently browser mode testing is written with `execa` + `npx vitest`, so I was keeping the same structure from:

https://github.com/vitest-dev/vitest/blob/c946b1c841c04f27f026c2b10b7b8f16ac12ceab/test/browser/specs/run-vitest.mjs#L12

The test runner used here is `node --test`, so test utils like `runVitest` are not available and `runner.test.mjs` would probably be easier to written with `execa`, so I'll keep that there for now.

This small test `custom-base.test.mjs` should be fine with `startVitest` API, so I'll rewrite that. Thanks for the suggestion!